pkg/exporter/sumologicexporter/exporter_test.go

```diff
@@ -406,254 +405,6 @@ func TestPushOTLPLogsClearTimestamp(t *testing.T) {
 	}
 }
 
-func TestPushOTLPLogs_AttributeTranslation(t *testing.T) {
-	createLogs := func() plog.Logs {
```

Comment:
In my opinion, the tests removed in file `exporter_test.go` are not very important and can be removed safely (because similar tests have been added to the processor), but please check if I'm right. The reason they are so complex is the fact that this feature couldn't be tested in an easier way while it has been in the exporter.